lib/index.js

```diff
@@ -80,6 +86,15 @@ Optimizely.prototype.track = function(track) {
   var revenue = track.revenue();
   if (revenue) payload.revenue = revenue *= 100;
   push(['trackEvent', track.event(), payload]);
+
+  var optimizelyClientInstance = window.optimizelyClientInstance;
+  if (optimizelyClientInstance && optimizelyClientInstance.track) {
+    var attributes = track.properties();
+    var userId = attributes.userId || track.userId();
```

Comment:
Hmmm @mikeng13 is the Fullstack JS SDK something you have to load synchronously? Can we load this async? Basically wondering if the Fullstack is essentially fully independent from the Optimizely X library. Because if it is, it might be better to create a new integration for Fullstack as a standalone integration. 

Is Optimizely treating it as a separate product? Because then I'd assume you guys would build more features for Fullstack so I think it'd be smart to separate it into a different integration so that we don't make the optimizely integration more and more complicated as we add support for more features in the future.

As far as the `track.userId()` being unknown after calling `.identify()`, I'm not sure how you are getting that. I can't reproduce it unfortunately :( and I'm pretty certain it's not broken, otherwise every Segment integration would be on fire right now. We might have to screenshare or something so you can show me.